tests/test_rules.py

```diff
@@ -79,7 +79,7 @@ def test_legal_trademark_violations(self):
     def test_optional_tags(self):
         files_with_incorrect_tags = []
         tags_pattern = re.compile(
-            r"cve\.\d+\.\d+|attack\.t\d+\.*\d*|attack\.[a-z_]+|car\.\d{4}-\d{2}-\d{3}")
+            r"cve\.\d+\.\d+|(attack\.t\d{4}\.\d{3}$|attack\.[gts]\d{4}$)|attack\.[a-z_]+|car\.\d{4}-\d{2}-\d{3}")
```

Comment:
```suggestion
            r"cve\.\d+\.\d+|attack\.(t\d{4}\.\d{3}|[gts]\d{4})$|attack\.[a-z_]+|car\.\d{4}-\d{2}-\d{3}")
```

Reply:
It's the same xD

Reply:
We removed one attack so we gain some bits :D